plugins/build-your-own.rst

```diff
@@ -0,0 +1,98 @@
+Building Custom Plugins
```

Comment:
no changes in this text, just moved out of the introduction as it was getting too long